examples/kubernetes/env.sh

```diff
@@ -6,16 +6,18 @@
 # use cases just need KUBECTL=kubectl, we'll make that the default.
 KUBECTL=${KUBECTL:-kubectl}
 
-# Kubernetes api address for $KUBECTL 
-# The default value is 127.0.0.1:8080
-# When the Kubernetes api server is not local, We can easily access the api by edit KUBERNETES_API_SERVER's value
-KUBERNETES_API_SERVER=${KUBERNETES_API_SERVER:-'127.0.0.1:8080'}
+# Kubernetes api address for $KUBECTL. When the Kubernetes api server is not
+# local, We can easily access the api by editing KUBERNETES_API_SERVER's value
+KUBERNETES_API_SERVER=${KUBERNETES_API_SERVER:-""}
 
 # Kubernetes namespace for Vitess and components.
 VITESS_NAME=${VITESS_NAME:-'default'}
 
 # Kubernetes options config
-KUBECTL_OPTIONS="--namespace=$VITESS_NAME --server=$KUBERNETES_API_SERVER"
+KUBECTL_OPTIONS="--namespace=$VITESS_NAME"
+if [[ -n $KUBERNETES_API_SERVER ]]; then
```

Comment:
nit: Please put the variable in quotes. This is recommended by the Bash Style guide: https://google.github.io/styleguide/shell.xml?showone=Testing_Strings#Testing_Strings

`if [[ -n "$KUBERNETES_API_SERVER" ]]; then`

This way we're also more consistent with the rest of the file.

The examples also use curly braces for the variable but I say let's not do that. That's just too much ;-)